src/mi/breakpoint.ts

```diff
@@ -49,8 +49,25 @@ export interface MIBreakListResponse extends MIResponse {
     };
 }
 
+export type MIBreakpointMode = 'hardware' | 'software';
+
 export interface MIBreakpointInsertOptions {
     temporary?: boolean;
+
+    /**
+     * The `mode` property is priotized over the `hardware` property. If `mode` is defined, then the information in the `hardware` flag is ignored during the insert breakpoint operation.
```

Comment:
prettier doesn't wrap/format comments, so my suggestion is just to wrap this very long line.

```suggestion
     * The `mode` property is priotized over the `hardware` property.
     * If `mode` is defined, then the information in the `hardware` flag is
     * ignored during the insert breakpoint operation.
```

Reply:
Update the comments.